addons/sale_management/report/sale_report_templates.xml

```diff
@@ -32,7 +32,8 @@
                                     t-att-style="option.discount and 'text-decoration: line-through' or None"
                                     t-att-class="option.discount and 'text-danger' or None"/>
                                 <div t-if="option.discount">
-                                    <t t-out="'%.2f' % ((1-option.discount / 100.0) * option.price_unit)"/>
+                                    <t t-set="discount_price" t-value="((1-option.discount / 100.0) * option.price_unit)"/>
```

Comment:
Why do you need a t-set here?

Reply:
@vava-odoo it's not needed. That was my first attempt until I realized I can shorten it to one line. I can make the change but then it's just the same as the other PR so I'm not sure which branch you would want to merge.